projects/libsass/build.sh

```diff
@@ -22,5 +22,5 @@ popd
 
 INSTALL_DIR="$SRC/libsass"
 
-$CXX $CXXFLAGS -I${INSTALL_DIR}/include -lFuzzingEngine data_context_fuzzer.cc \
-  -o $OUT/data_context_fuzzer ${INSTALL_DIR}/lib/libsass.a
+$CXX $CXXFLAGS -I${INSTALL_DIR}/include data_context_fuzzer.cc \
+  $LIB_FUZZING_ENGINE -o $OUT/data_context_fuzzer ${INSTALL_DIR}/lib/libsass.a
```

Comment:
Probably $LIB_FUZZING_ENGINE before data_context_fuzzer.cc

Reply:
I didn't do this because then we would need another line to avoid going over 80 chars.
Could you explain why you want these reordered?

Reply:
I thought order matter, saw in freeimage, jsonnet. if not, please skip.